oss-cloud-backend/services/databaseService.js

```diff
@@ -72,7 +72,22 @@ module.exports.addContributor = (contributor) => mysql
     ]]);
 
 
-module.exports.getAllContributions = () => mysql.query('SELECT * FROM contributions');
+module.exports.getContributionsPaging = (params) => mysql
+  .transaction()
+  .query(
+    `SELECT * FROM contributions ORDER BY
+    ${params.sortBy === 'repo' ? `owner ${params.sortDesc ? 'DESC' : 'ASC'}, ` : ''}

```

Comment:
What would the query look like if `params.sortBy === 'repo'` and `params.sortDesc` are both true?

Reply:
`SELECT * FROM contributions ORDER BY owner DESC, repo DESC LIMIT ?,? `